cv32e40x/tests/programs/custom/illegal_instr_test/illegal_instr_test.S

```diff
@@ -385,7 +366,6 @@ main:
 .word(0xfec76487)
 .word(0xbd1e312b)
 .word(0x700e3223)
-.word(0x612ed793)
```

Comment:
All of these removed .word() are bitmanip instructions, they have been moved into the test_bitmanip function